### DISCUSSION TITLE
how teleport gm to palyer's around ?

### DISCUSSION THREAD

**Discussion by ghost** [azerothcore/azerothcore-wotlk#13470] [General]
gps playername:   show the gm's gps.

**Comment by virgo77** [comment]
I don't understand what you want to do.
But maybe these commands may help:
- .appear nameofcharacter     //this command teleports the GM to a specified character
- .summon nameofcharacter  / this command teleports a character to the GM

Lists of GM Commands:
- https://www.azerothcore.org/wiki/gm-commands
- https://trinitycore.atlassian.net/wiki/spaces/tc/pages/2130065/GM+Commands

++
V.
